src/NzbDrone.Core/Tv/MovieService.cs

```diff
@@ -181,6 +187,31 @@ public Movie FindByTitle(string title, int year)
 
         public void DeleteMovie(int movieId, bool deleteFiles)
         {
+            //this next block was added in order to implement listsynccleaning
+            //start of block
+            if (deleteFiles)
```

Comment:
You shouldn't have to do that. Try debugging why the MovieDeletedEvent does not work.